lib/miq_automation_engine/service_models/miq_ae_service_authentication_rhsm.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceAuthenticationRhsm < MiqAeServiceAuthentication
```

Comment:
Oh now i see, ok so ill add service models to all the rest of the authentications.